README.md

```diff
@@ -190,6 +190,21 @@ Sent from the rover server to inform Mission Control of a joint's current positi
 - `joint` - the name of the joint
 - `position` - the current position in degrees
 
+## Set Inverse Kinematics
+### Description
+Sent from Mission Control to enable or disable inverse kinematics controls on the rover.
+
+### Syntax
+```
+{
+  type: "setInverseKinematics",
```

Comment:
Same thing, let's call this `setArmIKEnabled` or something, not `setInverseKinematics`